content/dev/dev-guide.md

```diff
@@ -0,0 +1,325 @@
+  - It mainly configures the appearance of an app's tile, the source of its glob (bundle of client resources like HTML, JS, and CSS files), and some additional metadata.
+    - We've given the app a `%title` of "Hello", which will be displayed on the app tile and will be the name of the app when others browse to install it. 
+    - We've given the app tile a `%color` of #8188C9, and specified the URL of an `%image` to display on the tile.
+    - The `%base` clause specifies the base URL path for the app. We've specified "hello" so it'll be `http://localhost:80/apps/hello/...` in the browser.
+      - **Note**: `%base` is only put into `/apps/[base]` here *because* the app is served as a glob through docket. A `%site` would be free to bind whereever.
+      - **Note**: `:80` is dependent on urbit being able to bind to that default. If it's in use by something else the port will likely be `:8080`, but you can see which one is in use by looking at the output generated in the terminal when you start your ship. Use the one your ship shows, this guide will assume the defaults are working for examples.
+    - For the glob, we've used a clause of `%glob-ames`, which means the glob will be served from a ship over Ames, as opposed to being served over HTTP with a `%glob-http` clause or having an Eyre binding with a `%site clause`. 
+    - Next we've specified `[~zod 0v0]`. Since `~zod` is the fakeship we'll install it on, the `%docket` agent will wait for a separate upload of the glob, so we can just specify `0v0` here as it'll get overwritten later.
+    - The `%version` clause specifies the version as a triple of major version, minor version and patch version. The rest is just some additional informative metadata which will be displayed in App Info.
+- With that saved we now have everything we need in the `hello` directory.
+
+##### Committing and Installing The Changes
+
+ - Let's start up fake zod.
+   - ```bash
+     cd ~/urbit-dev/
+     ./urbit zod
+     ```
+- Commit and install our changes.
+  - ```bash
+    |commit %hello
+    |install our %hello
+    ```
+  - You'll see `docket: awaiting manual glob for %hello desk` this is because our `desk.docket-0` files includes a `%glob-ames` clause which specifies our ship as the source, so it's waiting for us to upload the glob.
+- Open a browser and navigate to `http://localhost:80`. To get the login code, enter `+code` into the dojo prompt for your ship. In this case, the default is `lidlut-tabwed-pillex-ridrup`. You'll see our tile, but it says installing with a spinner due to the missing glob.
+
+##### Create Files for Glob
+
+We'll now create the files for the glob. We'll use a very simple static HTML page that just displayes "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal.
```

Comment:
```suggestion
We'll now create the files for the glob. We'll use a very simple static HTML page that just displays "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal.
```

Also what is "Eyre's channel system"? We don't mention it further. Should we link to docs about it? Alternatively just not bring it up.

Reply:
I'd avoid mentioning Eyre here.  Perhaps:

> Typically we'd have a more complex JS web app that talked to apps on our ship through the server API